tests/cli/test_main.py

```diff
@@ -21,7 +21,7 @@ def test_steal_config(
     monkeypatch: Any,
 ) -> None:
     patched_jobs_counts.return_value = {}
-    folder = shutil.move(nmrc_path, tmp_path / "orig-cfg")  # type: ignore
+    folder = shutil.move(str(nmrc_path), str(tmp_path / "orig-cfg"))
```

Comment:
Why do you need to convert paths to str? Does not `shutil.move()` support the path protocol?

Reply:
mypy had blamed that `move` requires `str`.
Perhaps it is a typeshed spike.
I did not investigate but quickly converted to the required type.
It is the test only, I wanted to make it pass quickly.

Reply:
I think typeshed's annotations are working around https://bugs.python.org/issue32689 (fixed in 3.9, reflected in typeshed).